packages/desktop/views/dashboard/wallet/views/implicit-account-creation-multistep/FundConfirmationView.svelte

```diff
@@ -48,7 +64,7 @@
             />
         </div>
         <Text type={TextType.h3} fontWeight={FontWeight.semibold}
-            >{localize('views.implicit-account-creation.steps.step2.view.title')} ({DUMMY_AMOUNT})</Text
+            >{localize('views.implicit-account-creation.steps.step2.view.title')} ({getOutputAmount()})</Text
```

Comment:
I don't think it matters here but, this will not be reactive

Reply:
Yes, it cannot modify the value of an output